presto-docs/src/main/sphinx/release/release-305.rst

```diff
@@ -0,0 +1,52 @@
+===========
+Release 305
+===========
+
+General Changes
+---------------
+
+* Fix failure of :doc:`/functions/regexp` for certain patterns and inputs
+  when using the default ``JONI`` library. (:issue:`350`)
+* Fix a rare ``ClassLoader`` related problem for plugins providing an ``EventListenerFactory``. (:issue:`299`)
+* Un-hide ``join_max_broadcast_table_size`` session property. (:issue:`346`)
+* Improve performance of queries when spill is enabled, but spill is not triggered. (:issue:`315`)
+* Improve memory cost estimations by calculating a plan's estimated peak memory usage. (:issue:`247`)
+* Include revocable memory in total memory stats. (:issue:`273`)
+* Add peak revocable memory to operator stats. (:issue:`273`)
+* Add :func:`ST_Points` function to access vertices of a linestring. (:issue:`316`)
+* Add a system table ``system.metadata.analyze_properties``
+  to list all :doc:`/sql/analyze` properties. (:issue:`376`)
+
+Resource Groups Changes
+-----------------------
+
+* Fix selection when selector uses pattern named capturing group as a variable. (:issue:`373`)
+
+Web UI Changes
+--------------
+
+* Display peak revocable memory, and current and peak total memory, in detailed query view. (:issue:`273`)
```

Comment:
Display peak revocable memory,  current total memory, and peak total memory, in detailed query view.

Reply:
I'd still make this change, but it is up to you.